pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -777,3 +784,12 @@ func parseCommitMsg(msg string, args argsTemplate) string {
 	}
 	return buf.String()
 }
+
+// getBranchName generates a new branch name in the format {eventName}-{uuid} if newBranch is true.
```

Comment:
Due to the description, the name should be `makeBranchName` or `buildBranchName` instead to follow the codebase convention.

Reply:
ok!!![renamed getBranchName to makeBranchName](https://github.com/pipe-cd/pipecd/pull/4395/commits/55a84657e639244de49dfcfac63253d8b7aafc62)